test/cases/coerced_tests.rb

```diff
@@ -759,6 +758,14 @@ def test_reverse_arel_assoc_order_with_function_coerced
     topics = Topic.order(Arel.sql("LEN(title)") => :asc).reverse_order
     assert_equal topics(:second).title, topics.first.title
   end
+
+  coerce_tests! :test_pluck_with_from_includes_quoted_original_table_name
```

Comment:
I'm curious why you're adding a new test along with fixing all the other tests?

Reply:
My mistake! This wasn't supposed to have been added.

> I'm curious why you're adding a new test along with fixing all the other tests?

Reply:
Yes, you can just make the change on your same local branch as this PR originally came from. When you push you local branch to GitHub, GitHub will automatically update the PR.

Thanks again for the great work.